app/templates/auth/caseworker/resident/question.hbs

```diff
@@ -22,6 +22,16 @@ as |page|
           data-test-question=true
         }}
       {{/form.group}}
+      {{#form.group
+        label="Anything to add?"
+      as |group|
+      }}
+        {{group.textarea
```

Comment:
Nice job adding in the additional note input box. Didn't we want it to show up on the profile page somewhere, though? It looks like right now the host and caseworker can enter an additional note, but it isn't displayed anywhere.

Reply:
Yes, it should be shown on the preview page. I planned to work on the preview page editing/styling on the weekend and have the page reviewed together. For next time, I will have all this up together for your review so you can do the test.

Reply:
Updated.